liquidity/liquidity.go

```diff
@@ -56,6 +102,21 @@ type Parameters struct {
 	// using it again.
 	FailureBackOff time.Duration
 
+	// MaximumPrepay is the maximum prepay amount we are willing to pay per
+	// swap.
+	MaximumPrepay btcutil.Amount
+
+	// MaximumSwapFeePPM is the maximum server fee we are willing to pay per
+	// swap expressed as parts per million of the swap volume.
+	MaximumSwapFeePPM int
+
+	// MaximumMinerFee is the maximum on chain fee we are willing to pay.
+	MaximumMinerFee btcutil.Amount
+
+	// ConfTarget is the number of blocks we aim to confirm our sweep
+	// transaction in. This value affects our miner fees.
+	ConfTarget int32
```

Comment:
I would expect the `ConfTarget` to be set on the suggested `LoopOutRequest` somewhere in this commit